bitnami/spring-cloud-dataflow/templates/_helpers.tpl

```diff
@@ -149,6 +149,17 @@ Return true if a configmap object should be created for Spring Cloud Skipper
 {{- end -}}
 {{- end -}}
 
+{{/*
+Return the database URL
+*/}}
+{{- define "scdf.database.url" -}}
+  {{- if .Values.externalDatabase.url }}
+    {{- printf "%s" .Values.externalDatabase.url -}}
+  {{- else -}}
+    {{- printf "" -}}
```

Comment:
Good idea, this cleaned things up. I had to create two methods for getting the jdbc URL : one for dataflow and then one for skipper because the database name (which is included in the URL when using mariadb) was different. This begs the question of if we should put the "url" property under both dataflow and skipper to be consistent?

Reply:
You are right. Since `url` is formed with `externalDatabase.xxx.database`, we should introduce both `externalDatabase .xxx.url`.